install.sh

```diff
@@ -293,6 +293,10 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_install_manpage() {
+    ln -s $PWD/nvm.1 /usr/local/share/man/man1/nvm.1
```

Comment:
I think mentioning brew was misleading. That path doesn't have anything to do with brew. That's actually one of the directories that man will search in for both a fresh install of macOS 10.13.3 and Ubuntu 16.04. The others are `/usr/share/man` (both), `/usr/X11/man` (macOS 10.13.3), and `/usr/local/man` (Ubuntu 16.04). So, ideally, `/usr/local/share/man` or `/usr/share/man` can be used since those both work on macOS and Ubuntu (I believe it should be the case for most/all Linux versions as well).

Reply:
In that case, the only remaining issue is that it's a path that requires sudo. Is there any way to modify the MANPATH so it can locate the file from something inside NVM_DIR?

Reply:
Actually, yes, I believe so. You can just set MANPATH to whatever and prepend or append a `:` to it, and man will add the other locations (listed above).

I'll add that tomorrow, depending on what you decide in the other ongoing comment.